redfish-core/lib/log_services.hpp

```diff
@@ -15,6 +15,8 @@
 */
 #pragma once
 
+#include "memory.hpp"
```

Comment:
Why include these? If we need to include these, lets move that function to a util

Reply:
I guess we are using them below for 

constexpr std::array<const char*, 2> processorInterfaces = {
    "xyz.openbmc_project.Inventory.Item.Cpu",
    "xyz.openbmc_project.Inventory.Item.Accelerator"}; 
    
    Will we ever have a  "xyz.openbmc_project.Inventory.Item.Accelerator" ?

Reply:
> I guess we are using them below for

constexpr std::array<const char*, 2> processorInterfaces = {
"xyz.openbmc_project.Inventory.Item.Cpu",
"xyz.openbmc_project.Inventory.Item.Accelerator"};

Yes, right. 

>  Will we ever have a  "xyz.openbmc_project.Inventory.Item.Accelerator" ?

I am not sure about the IBM system on this - It is defined in the existing processor.hpp so it will behave as like early